src/main/java/nc/students/ayaz/model/Video.java

```diff
@@ -1,18 +1,27 @@
-package nc.students.ayaz;
+package nc.students.ayaz.model;
 
 import cobertura.IgnoreDuringCodeCoverage;
 
 import java.util.ArrayList;
 import java.util.List;
 
-class Video {
+public class Video {
```

Comment:
Where do you use a "deserializer" ?
"serializer" is used to convert your object to JSON (and hence you need public getters), but I fail to find a place where you do the opposite (convert a JSON to an object).